### DISCUSSION TITLE
How to programatically grab image tags for each image?

### DISCUSSION THREAD

**Discussion by armondavani** [zanllp/infinite-image-browsing#749] [Q&A]
Hello,

I'm trying to run a python script that will grab the images that I "liked" for the day, and then run gens with their prompts, using various different settings. How would I look up an image's tags that I have applied to it?

It seems like this program uses a database of some kind, but I'm not familiar with Vue, nor python yet really, so would not mind some guidance and assistance here.

Thank you!

**Comment by zanllp** [comment]
You can retrieve it from this table in the database, but it does not have a field of the creation time.


![image](https://github.com/user-attachments/assets/7db6b4a9-5042-4754-8471-560c127a79be)
